examples/ex0_0.ipynb

```diff
@@ -200,7 +206,7 @@
    "source": [
     "import pandas as pd\n",
     "\n",
-    "df = pd.read_csv(\"../dat/recipes.csv\")\n",
+    "df = pd.read_csv(dirname(os.getcwd()) + \"/dat/recipes.csv\")\n",
```

Comment:
This is mostly just a style thing - but I tend to use `os.path.join` instead of handling paths as strings and relying on string operations.

Reply:
some OS (non-POSIX) may not be able to use plain relative paths. In notebooks in particular is better to use `getcwd`.

Reply:
I agree, that's not what I mean. I'm referring to using `os.path.join(dirname(os.getcwd()), "dat/recipes.csv")` as opposed to `dirname(os.getcwd()) + "/dat/recipes.csv"`.
Or, even better: Using `os.path.join(dirname(os.getcwd()), "dat", "recipes.csv")`.

Reply:
right, ideally we should use `pathlib.Path` but then the cell risks to become too verbose; so the implementation looked like a good compromise between fixing relative paths and not being too verbose.

Reply:
That is very understandable - they are examples, and indeed benefit from simplicity.